package/Dockerfile

```diff
@@ -1,8 +1,28 @@
-FROM ghcr.io/oracle/oraclelinux:7-slim
+FROM ghcr.io/verrazzano/golang:v1.20.12 as build
+
 RUN yum update -y \
     && yum clean all \
     && rm -rf /var/cache/yum/*
-COPY bin/backup-restore-operator /usr/bin/
+
+COPY . /go/src/github.com/backup-restore-operator/
+
+RUN dnf config-manager --enable ol8_appstream && \
```

Comment:
you can probably remove the dnf config-manager line and the install of git.  The previous version of this file does not appear to be installing git (unless you found it was being needed by the build script).

Reply:
I got the git not found command error so I added it.